src/Components/Home/contribution/ContributionSection.js

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import { Text, View, Image } from 'react-native';
+import styles from '../styles';
+import SectionHeader from '../../SectionHeader';
+import ContributionCard from './ContributionCard';
+
+
+function ContributionSection() {
+    const contributionCards = [
+        {
+            imageText: ["CODE"],
+            imageFile: 'code.png',
+            style: []  
+        },
+        {
+            imageText: ["DOCUMENTATION/","TRAINING"],
+            imageFile: 'documentation_training.png',
+            style: [[styles.cardMiddle]] 
+        },
+        {
+            imageText: ["DESIGN"],
+            imageFile: 'design_community.png',
+            style: [[styles.cardLast]] 
+        },
+        {
+            imageText: ["TESTING"],
+            imageFile: 'testing.png',
+            style: [] 
+        },
+        {
+            imageText: ["OUTREACH/RESEARCH"],
+            imageFile: 'research_outreach.png',
+            style: [[styles.cardMiddle]] 
+        },
+        {
+            imageText: ["PROJECT MANAGEMENT"],
+            imageFile: 'project_management.png',
+            style: [[styles.cardLast]] 
+        },
+
+    ]
+
+
+    return (
+        <View>
+        <SectionHeader title= "INTERESTED IN CONTRIBUTING?" />
+        <Text style={styles.description}>You can start contributing through one or more fields of your choice</Text>
+        <View style={styles.container}>
```

Comment:
if you do here `flexDirection:'row'` and `flexWrap:'wrap'` and  `justifyContent: 'space-between', or  justifyContent: 'space-around',` you would not have to have separate style for each card.

Reply:
I tried doing it with justifyContent and flex but since I need to maintain the left and right margin as 32, the design alignment shifts. Anything else I should try?

Reply:
well then apply padding to container above.

Reply:
@annabauza I've tried eliminating the styling for individual cards using the flex properties and padding as you suggested but it disrupts the alignment , especially the margins.  When I add the padding, the left margin increases.
![anitab-org](https://user-images.githubusercontent.com/48614937/93751941-8beb6d80-fc1b-11ea-89a7-0ede43a5ddf4.png)